content/user/agent/net/faq/DoesNotRecognizeThirdParty.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
This should probably be removed.  We do identify third party .NET libraries now.